sample-docker-app/Dockerfile.arm64v8

```diff
@@ -0,0 +1,12 @@
+FROM arm64v8/node:12.19.0-buster
+WORKDIR /usr/src/app
+
+RUN apt-get update && apt-get install -yq \
+  build-essential && \
+  apt-get clean && rm -rf /var/lib/apt/lists/*
+
```

Comment:
Maybe some image would be lighter than this one ?
`build-essential` part was needed to build some libraries for arm64.